README.md

```diff
@@ -5,65 +5,62 @@ Stores all the manifests and corresponding image-infos to test OSbuild.
 
 ### Update workflow
 
-The update workflow is made of 3 distinct steps:
+#### 1/ on GitHub: trigger the Gitlab CI
 
-#### First step, triggering the CI
+The update workflow creates a branch that is pushed to [this GitLab
+repository](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+Using a GitLab `trigger token` the GitHub action starts the CI with a
+different workflow than the one used to run the regression testing.
 
-The update workflow starts with an `Update DB` GitHub action that is manually
-triggered. This action imports a subset of the manifests from osbuild-composer,
-specifically those CI is able to build.
+#### 2/ on CI: Manifest generation
 
-Importing manifests which cannot be built by CI will leave the
-DB in a dirty state. For this reason they are excluded.
+The first job downloads the latest osbuild-composer sources from upstream,
+generates all test manifests using the gen-manifest tool and stores them as an
+artifact
 
-Once the import is done, a commit is created and pushed to [this GitLab repository](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+#### 3/ on CI: Image-info builds
 
-#### Second step, running the CI
+Then a job for each architecture and distribution listed in
+[.gitlab-ci.yml](https://github.com/osbuild/manifest-db/blob/main/.gitlab-ci.yml)
+will run in parallel. Each job will:
 
-Using a GitLab `trigger token` the GitHub action starts the CI with a
-different workflow than the one used to run the regression testing.
+- import the generated manifests
+- generate and include the image-info for the manifests where possible and
+  expose them as artifacts.
 
-A job for each architecture and distribution listed in
-[.gitlab-ci.yml](https://github.com/osbuild/manifest-db/blob/main/.gitlab-ci.yml)
-will run in parallel. Each job will generate and include the image-info for the
-manifests where possible and expose them as artifacts. As a final step, a
-conclusion job downloads every artifact and includes it in the commit that
-initially triggered the CI job.
+#### 4/ on CI: PR creation
 
-#### Third step, opening a PR on GitHub
+Finally a last job imports the generated manifests and image-info, updates the
```

Comment:
I believe that *last* is always used with *the*. Therefore, I suggest changing *a last job* to *the last job*.